test/test_helper.rb

```diff
@@ -24,7 +24,7 @@
 require 'minitest/autorun'
 require 'minitest/stub/const'
 require 'webmock/minitest'
-require 'mocha/mini_test'
+require 'mocha/minitest'
```

Comment:
Please bump https://github.com/Shopify/kubernetes-deploy/blob/master/kubernetes-deploy.gemspec#L39 to 1.5.0. This blows up pre that version https://github.com/freerange/mocha/commit/55d66ed4f5e6cad95e1605341ec3c608be2c6833